packages/shared/components/inputs/AssetAmountInput.svelte

```diff
@@ -105,7 +106,7 @@
         {#if asset}
             <button on:click={onClickAvailableBalance}>
                 <Text color="gray-600" darkColor="gray-500" fontSize="xs" classes="cursor-pointer">
-                    {localize('general.availableBalance', {
+                    {localize('general.balance', {
```

Comment:
This doesn't take a value so it is not showing the actual amount next to the word, we might need to just use the general.availableBalance but just change the string. or have a specific locale for here.

Reply:
Didn't catch this at first, now it is fixed, thank you :pray: